pkg/api/controllers/usercontrollers/usercontroller.go

```diff
@@ -144,7 +145,13 @@ func (u *userControllersController) amOwner(peers tpeermanager.Peers, cluster *v
 	scaled := int(ck) * len(peers.IDs) / math.MaxUint32
 	logrus.Debugf("%s(%v): (%v * %v) / %v = %v[%v] = %v, self = %v\n", cluster.Name, cluster.UID, ck,
 		uint32(len(peers.IDs)), math.MaxUint32, peers.IDs, scaled, peers.IDs[scaled], peers.SelfID)
-	return peers.IDs[scaled] == peers.SelfID
+	isOwner := peers.IDs[scaled] == peers.SelfID
+	if isOwner {
+		metrics.SetClusterOwner(peers.SelfID, cluster.Name)
```

Comment:
fine leaving this here but seems weird to me that we'd set the metrics cluster owner in a bool check function. maybe better logic in peersSync that calls amOwner?

Reply:
Good call. Moved.